src/declarations/stencil-public-runtime.ts

```diff
@@ -512,6 +523,7 @@ export declare const Host: FunctionalComponent<HostAttributes>;
  */
 export declare const Fragment: FunctionalComponent<{}>;
 
+// eslint-disable-next-line jsdoc/require-param -- we don't want to JSDoc these overloads at this time
```

Comment:
I didn't find an elegant way to JSDoc these overloads, and TBH I don't know if we even _want_ to for this namespace